### PR TITLE
Adding note for list channel api

### PR DESCRIPTION
**Pull request by kallu800**

### CODE REVIEW

**Review by Lauragra — APPROVED** [review]

**Review by sumitgupta3 — APPROVED** [review]

File: api-reference/beta/api/channel-list.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ Namespace: microsoft.graph
 
 Retrieve the list of [channels](../resources/channel.md) in this [team](../resources/team.md).
 
+>**Note:** Guest users who are not a part of private/shared channel will not be able to see the same in the response for this API.
```
Comment (Danielabom) [inline api-reference/beta/api/channel-list.md:18]:
```suggestion
>**Note:** Guest users who are not a part of private/shared channels are not able to see the same in the response for this API.
```

File: api-reference/v1.0/api/channel-list.md [hunk 2]
```diff
@@ -14,6 +14,8 @@ Namespace: microsoft.graph
 
 Retrieve the list of [channels](../resources/channel.md) in this [team](../resources/team.md).
 
+>**Note:** Guest users who are not a part of private/shared channel will not be able to see the same in the response for this API.
```
Comment (Danielabom) [inline api-reference/v1.0/api/channel-list.md:17]:
```suggestion
>**Note:** Guest users who are not a part of private/shared channels are not able to see the same in the response for this API.
```

**Review by Danielabom — APPROVED** [review]
Editorial update. PR approved.

File: api-reference/beta/api/channel-list.md [hunk 3]
```diff
@@ -15,7 +15,9 @@ Namespace: microsoft.graph
 
 Retrieve the list of [channels](../resources/channel.md) in this [team](../resources/team.md).
 
-This method supports federation. Any shared channel that the request initiator belongs to will be included in the response.
+sokallan_listChannel
```
Comment (Lauragra) [inline api-reference/beta/api/channel-list.md:18]: Is this stray text that should be deleted?

**Pull request merged**